west-test.yml

```diff
@@ -0,0 +1,8 @@
+manifest:
+  version: "0.12"
+
+  self:
+    west-commands: scripts/west-commands.yml
+    import:
+      - submanifests
```

Comment:
The west-test.yml only exists, because west does not support a selective import.
When west implements such a feature, I'd like to merge the west and west-test yml files.
Which then would result in two different folders to be imported from one west file. Possible leading to confusion.
Also in all the internal repos there will a submanifest folder as well. And I'd like to keep it consistent.

Reply:
> When west implements such a feature, I'd like to merge the west and west-test yml files.
> Which then would result in two different folders to be imported from one west file.

Even when `west` supports selective imports, then I believe having the option of both `submanifests` and `test-manifests` makes a lot of sense.
There is a chance that someone placing a drop-in manifest in `submanifests` don't want to also load test specific `99-default-test-nrf.yml` manifest.

Drop-in manifests are very useful if you want to adjust the projects without having to do local modifications to the repo itself.

So even when west supports selective imports, then in `west.yml` there could be:
```
 self:
    import:
      - submanifests
      - test-manifests
```
where `test-manifests` could be selectively enabled and `submanifests` foolder available to users for drop-in manifests.
This allows selective load of `submanifests` without having to load `test` specific manifests also.

Reply:
this change sounds good to me.

Reply:
Renamed the submanifest folder to test-manifests.